rest-api-spec/src/main/resources/rest-api-spec/api/cluster.delete_decommission_awareness.json

```diff
@@ -0,0 +1,19 @@
+{
+  "cluster.delete_decommission_awareness": {
+    "documentation": {
+      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
+      "description": "Delete any existing decommission."
+    },
+    "stability": "experimental",
```

Comment:
why is it experimental?

Reply:
We will be marking this to stable once all meta issues are resolved we already have task identified for same.
https://github.com/opensearch-project/OpenSearch/issues/3957